datrie/tail.c

```diff
@@ -251,6 +251,57 @@ tail_fwrite (const Tail *t, FILE *file)
 }
 
 
+size_t
+get_tail_serialized_size (const Tail *t)
+{
+    size_t static_count = (
+        sizeof(int32) * 3 // TAIL_SIGNATURE , t->first_free , t->num_tails
+    );
+    size_t dynamic_count = 0u;
+    if(t->num_tails > 0){
+        dynamic_count += (
+            4 * 2 // t->tails[i].next_free , t->tails[i].data
+            + 2 // length
+        ) * t->num_tails;
+        for (TrieIndex   i = 0; i < t->num_tails; i++) {
+            if (t->tails[i].suffix > 0)
+            {
+                dynamic_count += strlen ((const char *)t->tails[i].suffix) * sizeof(const char);
```

Comment:
Please use `trie_cher_strlen (t->tails[i].suffix)` (newly added in `"trie-string.h"` in master) instead of `strlen()` here. And you may also use `sizeof (TrieChar)` instead of `sizeof(const char)`.

Reply:
Fixed.